### PR TITLE
Rgbank replace apache with nginx

### PR DESCRIPTION
**Pull request by caseylondon**
This pull request introduces two new profile:
- profile::apache::remove - Uses the apache module to ensure apache is absent from a system
- profile::nginx - Uses the puppet/nginx module to manage nginx

The role::appserver role has been updated to use the two new profiles to swap out nginx for apache

Further, a specific version of the rgbank module is being pulled since it has the required nginx changes for the web app component.

### CODE REVIEW

File: applications.yaml [hunk 1]
```diff
@@ -5,6 +5,12 @@ dev:
       components:
         ['Rgbank::Web','Rgbank::Db', 'Rgbank::Load']:
           - rgbank-dev.vm
+rgbank_replace_apache_with_nginx:
```
Comment (ccaum) [inline applications.yaml:8]: Do you plan to leave this in permanently?

Comment (caseylondon) [reply applications.yaml:8]: No. Once the change is pushed through to production I'll rip this code out.

Comment (ccaum) [reply applications.yaml:8]: Sounds good. Thanks!

**Comment by caseylondon** [comment]
All unit and acceptance tests are passing. See Jenkins job 59

**Pull request merged**